ui/apps/platform/src/Containers/Policies/Detail/PolicyDetail.tsx

```diff
@@ -275,14 +276,25 @@ function PolicyDetail({
                 </AlertGroup>
             </PageSection>
             <ConfirmationModal
+                title={isExternalPolicy(policy) ? 'Delete policy?' : 'Permanently delete policy?'}
```

Comment:
Observations for general understanding, but not change requests for this contribution:
* Conditional title of modal is difficult for technical documentation.
* Although it seems necessary in this situation, it is a warning sign why UI even allows the situation.

Reply:
- I'm on board with only displaying "Delete policy?" both here and on the other page. Up until now there was no title at all so the distinction between permanent deletion (the expected behavior for apps that allow deletion) and "temporary resync deletion" is probably not as necessary, especially given the text in the body of the modal.
- Agreed. I'm hoping that the need or desire to delete **external** policies is off the intended path enough that almost zero users will encounter the difference in behavior.